package.json

```diff
@@ -12,118 +12,115 @@
-    "fork-ts-checker-webpack-plugin": "0.4.1",
+    "codelyzer": "5.2.0",
+    "copy-webpack-plugin": "5.1.1",
+    "css-loader": "3.3.2",
+    "eslint": "6.7.2",
+    "eslint-config-jhipster": "0.0.1",
+    "eslint-config-prettier": "6.7.0",
+    "eslint-loader": "3.0.3",
+    "file-loader": "5.0.2",
     "friendly-errors-webpack-plugin": "1.7.0",
     "generator-jhipster": "6.9.0",
     "html-loader": "0.5.5",
     "html-webpack-plugin": "3.2.0",
-    "husky": "0.14.3",
-    "jest": "22.4.3",
-    "jest-preset-angular": "5.2.2",
+    "husky": "3.1.0",
+    "jest": "24.9.0",
+    "jest-date-mock": "1.0.7",
+    "jest-junit": "10.0.0",
+    "jest-preset-angular": "8.0.0",
     "jest-sonar-reporter": "2.0.0",
-    "lint-staged": "7.0.0",
-    "moment-locales-webpack-plugin": "1.0.5",
-    "node-sass": "^4.12.0",
-    "postcss-loader": "2.1.1",
-    "prettier": "1.11.1",
-    "proxy-middleware": "0.15.0",
-    "raw-loader": "0.5.1",
-    "rimraf": "2.6.1",
-    "sass-loader": "7.0.1",
+    "lint-staged": "8.2.1",
+    "mini-css-extract-plugin": "0.8.0",
+    "moment-locales-webpack-plugin": "1.1.2",
+    "optimize-css-assets-webpack-plugin": "5.0.3",
+    "postcss-loader": "3.0.0",
+    "prettier": "1.19.1",
+    "reflect-metadata": "0.1.13",
+    "rimraf": "3.0.0",
+    "sass": "1.23.7",
+    "sass-loader": "8.0.0",
     "simple-progress-webpack-plugin": "1.1.2",
-    "style-loader": "0.20.3",
-    "tapable": "1.0.0",
-    "thread-loader": "1.1.5",
-    "to-string-loader": "1.1.5",
-    "ts-loader": "4.0.1",
-    "tslint": "5.9.1",
-    "tslint-config-prettier": "1.9.0",
-    "tslint-loader": "3.6.0",
-    "typescript": "2.7.2",
-    "uglifyjs-webpack-plugin": "1.2.5",
-    "webpack": "4.8.0",
-    "webpack-cli": "2.1.3",
-    "webpack-dev-server": "3.1.11",
-    "webpack-merge": "4.1.2",
-    "webpack-notifier": "1.6.0",
-    "webpack-visualizer-plugin": "0.1.11",
-    "workbox-webpack-plugin": "3.2.0",
-    "write-file-webpack-plugin": "4.2.0",
-    "xml2js": "0.4.19"
+    "style-loader": "1.0.1",
+    "terser-webpack-plugin": "2.3.0",
+    "thread-loader": "2.1.3",
+    "to-string-loader": "1.1.6",
+    "ts-loader": "6.2.1",
+    "tslint": "6.0.0",
+    "typescript": "3.7.5",
+    "webpack": "4.41.2",
+    "webpack-bundle-analyzer": "3.6.0",
+    "webpack-cli": "3.3.10",
+    "webpack-dev-server": "3.9.0",
+    "webpack-merge": "4.2.2",
+    "webpack-notifier": "1.8.0",
+    "workbox-webpack-plugin": "4.3.1",
+    "write-file-webpack-plugin": "4.5.1"
   },
   "engines": {
     "node": ">=8.9.0",
-    "yarn": ">=1.3.2"
-  },
-  "lint-staged": {
-    "src/**/*.{ts,css,scss}": [
-      "prettier --write",
-      "git add"
-    ]
+    "yarn": ">=1.19.0"
   },
   "scripts": {
-    "precommit": "lint-staged",
-    "prettier:format": "yarn prettier --write 'src/**/*.{ts,css,scss}'",
-    "lint": "tslint --project tsconfig.json -e 'node_modules/**'",
+    "prettier:format": "prettier --write \"{,src/**/}*.{md,json,ts,css,scss,yml,java}\"",
```

Comment:
@pascalgrimaud that fix went to 6.9.0 only

Reply:
I guess I was thinking about applying prettier on java classes, but, later backed out as it would increase the change area. I will revert for now and we can do this in a separate PR.

Reply:
reverted this change